api/v1beta1/disruption_cron_webhook.go

```diff
@@ -196,3 +203,36 @@ func (d *DisruptionCron) validateDisruptionCronSpec() error {
 
 	return nil
 }
+
+func (d *DisruptionCron) validateMinimumFrequency(minFrequency time.Duration) error {
+	schedule, err := cron.ParseStandard(d.Spec.Schedule)
+	if err != nil {
+		return fmt.Errorf("spec.Schedule must follow the standard cron syntax: %w", err)
+	}
+
+	specDuration := defaultDuration
+	if d.Spec.DisruptionTemplate.Duration.Duration() > 0 {
```

Comment:
To cover this case, can you add this test in the `success cases` of the `ValidateCreate` method:

``` golang
When("the disruption template duration is greater than 0", func() {
    It("should validate the disruption cron successfully", func() {
	    // Arrange
	    disruptionCron := makeValidDisruptionCron()
	    disruptionCron.Spec.DisruptionTemplate.Duration = "1s"
    
	    disruptionCronJSON, err := json.Marshal(disruptionCron)
	    Expect(err).ShouldNot(HaveOccurred())
    
	    expectedAnnotation := map[string]string{
		    EventDisruptionCronAnnotation: string(disruptionCronJSON),
	    }
    
	    By("sending the EventDisruptionCronCreated event to the broadcast")
	    mockEventRecorder := mocks.NewEventRecorderMock(GinkgoT())
	    mockEventRecorder.EXPECT().
		    AnnotatedEventf(
			    disruptionCron,
			    expectedAnnotation,
			    Events[EventDisruptionCronCreated].Type,
			    string(EventDisruptionCronCreated),
			    Events[EventDisruptionCronCreated].OnDisruptionTemplateMessage,
		    )
    
	    disruptionCronWebhookRecorder = mockEventRecorder
    
	    // Act
	    warnings, err := disruptionCron.ValidateCreate()
    
	    // Assert
	    Expect(warnings).To(BeNil())
	    Expect(err).ShouldNot(HaveOccurred())
    })
})
```